go/libkb/chain_link_v2.go

```diff
@@ -67,6 +67,29 @@ func (t SigchainV2Type) IsTeamType() bool {
 	}
 }
 
+func (t SigchainV2Type) TeamAllowStubWithAdminFlag(isAdmin bool) bool {
+	role := keybase1.TeamRole_READER
+	if isAdmin {
+		role = keybase1.TeamRole_ADMIN
+	}
+	return t.TeamAllowStub(role)
+}
+
+func (t SigchainV2Type) RequiresAdminPermission() bool {
+	switch t {
+	case SigchainV2TypeTeamNewSubteam,
+		SigchainV2TypeTeamChangeMembership,
+		SigchainV2TypeTeamLeave,
+		SigchainV2TypeTeamSubteamHead,
+		SigchainV2TypeTeamRenameSubteam,
+		SigchainV2TypeTeamInvite,
+		SigchainV2TypeTeamRenameUpPointer:
+		return true
+	default:
+		return false
+	}
```

Comment:
It might be safer to invert this and whitelist the types that _don't_ require admin permission. That way we don't create security problems when we forget to update this function?